src/components/js/EcApplication.js

```diff
@@ -58,54 +158,39 @@ export default {
           this.$message.error('Não foi possível instalar o aplicativo', 3)
         })
     },
-    deleteApp () {
-      this.ecomApps.removeApplication(this.localApplication._id).then(() => {
-        this.isInstalled = false
-      })
-    },
-    toggleSettings () {
-      this.showSettings = (!this.showSettings)
-    }
-  },
-  computed: {
-    appId () {
-      return this.applicationBody.app_id
-    },
-    icon () {
-      return this.applicationBody.icon
-    },
-    title () {
-      return this.applicationBody.title
-    },
-    category () {
-      return this.applicationBody.category
-    },
-    author () {
-      return this.applicationBody.partner.name
-    },
-    shortDescription () {
-      return this.applicationBody.short_description
+
+    uninstallApp () {
+      this.ecomApps.removeApplication(this.localApplication._id)
     },
-    description () {
-      return this.applicationBody.description
+
+    handleTabChange (key, type) {
+      this[type] = key
     },
-    localApplication: {
-      get () {
-        return this.applicationBody
-      },
-      set (data) {
-        this.applicationBody = data
-        this.$emit('update:application', data)
+
+    updateTabContent () {
+      const { applicationBody } = this
+      if (applicationBody.app_id && !applicationBody.author_id) {
+        this.fetchMarketApplication()
+      }
+      if (applicationBody._id && !applicationBody.admin_settings) {
+        this.fetchStoreApplication(applicationBody._id)
       }
     }
   },
-  created () {
-    const { applicationBody } = this
-    if (applicationBody.app_id && !applicationBody.partner) {
-      this.fetchMarketApplication()
-    }
-    if (applicationBody._id && !applicationBody.admin_settings) {
-      this.fetchStoreApplication(applicationBody._id)
+
+  watch: {
+    activeTabKey: {
+      handler () {
+        if (this.activeTabKey === 'related') {
+          this.findRelateds()
+        } else {
+          this.updateTabContent()
+        }
+      },
+      immediate: true
+    },
+    localApplication () {
+      this.loading = true
```

Comment:
Não entendi porque você seta o loading aqui :thinking: 
Sempre que o `localApplication` for alterado você seta o loading, e depois ele não é resetado em lugar nenhum (pelo menos eu não vi), isto não vai funcionar como deveria, vai?
Além disto, você não precisa de watch pra este computed, só usar a função `set` dele que já está declarada.

Reply:
Flws!